src/index.html

```diff
@@ -4,9 +4,369 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home">
+        <img class="header__logo" src="./images/logo.png"
+        alt="moyo">
+      </a>
+      <nav>
+        <ul class="nav">
+          <li class="nav__item">
+            <a href="#" class="is-active nav__link">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link"
+            data-qa="nav-hover">Laptops & computers </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Gadgets</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Tablets</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Photo</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div class="cards">
+        <div
+         class="card cards__card"
+         data-qa="card"
+        >
+          <img
+           src="./images/imac.jpeg"
+           alt="Apple A1419"
+           class="card__image"
+          >
+          <h2 class="card__name">
+           APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <p class="card__product-code">
+           Product code: 195434
+          </p>
+          <div class="stars">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p class="stars__reviews">
+              Reviews: 5
+            </p>
+          </div>
+          <div class="card__price">
+            <p class="card__price-text">Price:</p>
+            <p class="card__price-cost">$2,199</p>
+          </div>
+          <a
+           href="#"
+           class="card__button"
+           data-qa="hover"
+          >
+           Buy
+          </a>
+        </div>
+
+        <div
+         class="card cards__card"
+         data-qa="card"
+        >
+          <img
+           src="./images/imac.jpeg"
+           alt="Apple A1419"
+           class="card__image"
+          >
```

Comment:
Add 2 spaces everywhere
```suggestion
          <img
            src="./images/imac.jpeg"
            alt="Apple A1419"
            class="card__image"
          >
```